app/controllers/health_checks_controller.rb

```diff
@@ -8,9 +8,24 @@ class HealthChecksController < ActionController::Base
   newrelic_ignore_apdex
 
   def show
+    migrations = migration_status
     body = {
       healthy: true
-    }.merge(Rails.application.config.build_version || {})
+    }.merge(Rails.application.config.build_version || {}).merge(migrations)
```

Comment:
```suggestion
    }.merge(Rails.application.config.build_version || {}).merge(migration_status)
```
and line 11 can be deleted